documentation/cron-jobs.md

```diff
@@ -0,0 +1,9 @@
+# Cron jobs
```

Comment:
This documentation is in `govuk-paas.md` I think

Reply:
Yep, I'll remove the file.